src/app/authorize/components/oauth-error/oauth-error.component.html

```diff
@@ -123,8 +123,8 @@
 
       <div i18n="@@authorize.error">
         You have reached this page due to an error with the application's
-        integration. Please report this error using the "help" button below or
-        contact the organization requesting your ORCID iD.
+        integration. Please report this error to the organization that is
```

Comment:
@Camelia-Orcid Could you also update the `authorize.error` property to be the same text in the template and in the property

Reply:
Fixed.